src/benchmarks/micro/libraries/System.Text.Json/Serializer/ReadParameterizedConstructor.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using MicroBenchmarks.Serializers;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    [GenericTypeArguments(typeof(SimpleStructWithProperties), typeof(SimpleStructWithProperties_Immutable), typeof(SimpleStructWithProperties_1Arg))]
+    [GenericTypeArguments(typeof(LoginViewModel), typeof(Parameterized_LoginViewModel_Immutable), typeof(Parameterized_LoginViewModel_2Args))]
+    [GenericTypeArguments(typeof(Location), typeof(Parameterized_Location_Immutable), typeof(Parameterized_Location_5Args))]
+    [GenericTypeArguments(typeof(IndexViewModel), typeof(Parameterized_IndexViewModel_Immutable), typeof(Parameterized_IndexViewModel_2Args))]
+    [GenericTypeArguments(typeof(MyEventsListerViewModel), typeof(Parameterized_MyEventsListerViewModel_Immutable), typeof(Parameterized_MyEventsListerViewModel_2Args))]
+    [GenericTypeArguments(typeof(Parameterless_Point), typeof(Parameterized_Point_Immutable), typeof(Parameterized_Point_1Arg))]
+    [GenericTypeArguments(typeof(Parameterless_ClassWithPrimitives), typeof(Parameterized_ClassWithPrimitives_Immutable), typeof(Parameterized_ClassWithPrimitives_4Args))]
+    public class ReadParameterizedConstructor<TTypeWithParameterlessCtor, TTypeWithParameterizedCtor1, TTypeWithParameterizedCtorType2>
```

Comment:
I need to wrap my head around it and figure a way we could shorten the benchmark name, because as of today it would be something like:

`System.Text.Json.Serialization.Tests.ReadParameterizedConstructor<SimpleStructWithProperties, SimpleStructWithProperties_Immutable, SimpleStructWithProperties_1Arg>.Deserialize_Parameterless`

Which would make hard to read the results both from console and the Reporting System